### ISSUE TITLE
Final Fantasy XIV Free Trial (312060)

### ISSUE THREAD

**Issue by neirenoir** [ValveSoftware/Proton#1162]
I don't know if this should be posted in the FFXIV Full Version (39210) issue, since the client and the launcher are almost identical, but I will open a new issue for correctness. Merge it if you think it doesn't deserve its own issue.

Whenever I launch the launcher, it does not seem to detect my DXVK, and it simply greys out the DirectX 11 switch in the options. I am only able to launch it in DirectX 9.0c mode. 

### Things I tried
- Used Proton 3.7-3 and 3.7-5 Beta. Neither of those work.
- Reinstalled the game.
- Used winetricks to install xact and/or set d3d11 and dxgi to native.
- Used /tmp/proton_run to run the launcher outside Steam. More info on this later.
- **Launched the game with PROTON_USE_WINED3D11=1. The switch now works**, but the game crashes on launching, so it is not what I actually want.
- Opening Steam from CLI and watching the output. The CLI seems to be mute in this.

Whenever I use /tmp/proton_run to launch the game outside of Steam, two things happen:
- It spews some stuff to the console about not being able to initialize OpenVR, 'failed to create Vulkan instance', and 'extension "NV-GLX" missing on display ":0"' (which is weird, considering I do not even have an Nvidia card here, and this error is apparently related to Nvidia's proprietary drivers, from what I have seen). You can find a sample of the console output [here](https://gist.github.com/shirona/ed95a3a3688c2c7395e5a4d9531c5fb1).
- Setting the PROTON_USE_WINED3D11=1 environment variable seems not to enable the switch, unlike when it is run from Steam.

### Things to note about the game's system detection tool
The game comes with a tool to obtain information about your system, running processes, and other things that can help with technical support, similar to Steam's system report. 
- It lists my OS as Windows XP Professional, despite the Proton prefix being set to Windows 7.
- It lists the DirectX version of the system as "DirectX 9.0c (4.09.0000.0904)", even when the switch has been enabled by using the WINED3D11 flag.
- It lists my GPU as a Radeon R9 285, even though it is a R9 380.
- It detects the GPU driver as aticfx.dll
- The install information section just says "Wine Gecko (32-bit)". Can confirm the prefix is actually 64-bit, according to winetricks.
- Nothing seems to change between the WINED3D11 version and the DXVK version.

### System info
- AMD FX-8350 CPU
- AMD R9 380 GPU
- Debian Sid, updated just today
- Kernel 4.17.0-3
- Mesa 18.1.7
- i3 window manager
- Running inside firejail. Wanna note that no other game seems to have a problem with this, just in case. Can post sandbox profile if necessary.

### Other info
- The game is confirmed to work flawlessly in other users' systems, after installing some stuff with winetricks.
- Even before installing xact and tweaking the prefix, users report they can actually try to launch the game in DX11 mode, but it crashes. Not in my case, I can't even attempt to run it in DX11 mode, and I suspect it would actually run if it could.
- I can confirm DXVK works in my system. Tried it with Slap City, and even though the characters appear invisible, the DXVK_HUD was showing, so I know it was running well.
- These are the Proton logs for [running without PROTON_USE_WINED3D11=1](https://gist.github.com/shirona/c9bd9d75ce79b9161bc0b288263ef3fc), and for [running with PROTON_USE_WINED3D11=1](https://gist.github.com/shirona/91eaefe87c654ba54130bb549453013b)
- Setting the d3d11 and dxgi libraries to native makes the game not enable the switch even if PROTON_USE_WINED3D11=1 is passed.

All in all, it appears my DXVK isn't reporting itself correctly to FFXIV in my system, which makes it think it is not capable of running DX11 games, when it is. But why is WINED3D11 reporting itself correctly, and not DXVK?

**Comment by neirenoir** [comment]
I have recently tried to run Steam without a sandbox, and the results were the same. I can confirm this is not a Firejail issue.

**Comment by XakepSDK** [comment]
https://gist.github.com/XakepSDK/a660a77f5ce42c0a6168e3da14b7498a
Launcher says HTTPS System Error (404)
This happens only after first launch

**Comment by nstgc** [comment]
Literally nothing works, not even the launcher. Not even the Proton apparently. The prefix isn't even properly set up.

System info: https://gist.github.com/nstgc/3b18051f7c19e88680b56b41976bd81d

Proton Log output:

```
======================
Proton: 1569267010 proton-4.11-6
SteamGameId: 312060
Command: ['/btrfs/raid0/games/win32/steam/Steam/steamapps/common/FINAL FANTASY XIV Online/boot/ffxivboot.exe', '-issteamfreetrial']
Options: set()
======================
```

That's it. That's everything. More over, consider the following output of `ls **` from inside the compat director: https://gist.github.com/nstgc/6878230dfd0694ad542fb18d1407610d

**Comment by XakepSDK** [comment]
Use proton-ge or proton-tkg
@nstgc

**Comment by nstgc** [comment]
> Use proton-ge or proton-tkg
> @nstgc

Steam should at least be able to set up a prefix on its own. Its not even doing that which leads me to believe there is something wrong with Steam or the FF14 Demo's repo.

As for troubleshooting I might have done, I've tried all other official Proton builds.

**Comment by nstgc** [comment]
What I did was run
``WINEPREFIX="path/to/compat/312060/pfx" "/full/path/to/Proton 4.11/dist/bin/wine" winecfg``
to set up a prefix. It now runs correctly. I don't think this is a problem with this game specifically

@kisak-valve Should I create a new issue for this?

**Comment by dlm21** [comment]
I spent like 3 days trying to figure out this issue on a fairly new Solus install. Initially it would crash when clicking on the "play dx9" button, and after all my fixes, wouldn't even get the launcher off the ground. I did everything from clearing the proton prefixes, stracing every call, running the commands with the same environment vars, etc, to removing the game and all traces of steam completely and starting over, but I never got the game running, nor did it allow DX11 to be enabled. The steam log showed it couldn't find my video card (?) despite every other game working fine. I tried Lutris with the same GE proton build and it was able to show DX11, but since my free trial account is tied to steam, was unable to proceed past the launcher.

My solution - I installed Garuda Linux and it worked fine the first time (browser/cutscrene fixes included), DX11 and all. I hate not finding a solution for the actual problem, but I really tried everything I could find online from the past year or so. So if nothing else, I can at least confirm the free trial on steam works great (using proton 6.16-GE-1), when your system's personal stars all align :)

**Comment by OlivierFRDierick** [comment]
Hello,

I've installed the Trial version of Final Fantasy XIV on Steam (Linux client with compatibility forced to Proton Experimental).
I use the default launcher. I can't agree to the Terms of Service as the 'Agree and Continue' button stays grayed out after scrolling the text down and checking the 'Agree to the Terms of Service' checkbox.

Regards.
